TELCODOCS-609: D/S Docs & RN: TELCODOCS-579 Remote worker node updates (#61800)
openshift/openshift-docs: modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc

// This module is included in the following assemblies: 
//
// installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

:_content-type: PROCEDURE
[id="configuring-host-network-interfaces-for-subnets_{context}"]
= Optional: Configuring host network interfaces for subnets

For edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. This often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Such a setup can reduce latency for the edge and allow for enhanced scalability.

> If you have established different network segments or subnets for remote worker nodes as described in the section on "Establishing communication between subnets", you must specify the subnets in the `machineNetwork` configuration setting if the workers are using static IP addresses, bonds or other advanced networking. When setting the node IP address in the `networkConfig` paramter for each remote worker node, you must also specify the gateway and the DNS server for the subnet containing the control plane nodes when using static IP addresses. This ensures the remote worker nodes can reach the subnet containing the control plane nodes and that they can receive network traffic from the control plane.

Author: That's how it was originally. I was asked to add the first paragraph in the peer review.

// This module is included in the following assemblies: 
//
// installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

:_content-type: PROCEDURE
[id="ipi-install-configuring-host-network-interfaces-for-subnets_{context}"]
= Configuring host network interfaces for subnets

> For edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. Locating remote worker nodes in subnets often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Setting up subnets for edge computing scenarios can reduce latency for the edge and allow for enhanced scalability.

Reviewer: ```suggestion
For edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. To locate remote worker nodes in subnets, you might use different network segments or subnets for the remote worker nodes than you used for the control plane subnet and local worker nodes. You can reduce latency for the edge and allow for enhanced scalability by setting up subnets for edge computing scenarios.
```

Reviewer: I feel like I stumbled through these two sentences. I wonder if they can be re-written a little to keep the user actions in focus. Starting both sentences with a gerund in a row makes for some loaded reading and its hard to keep all the details straight I think. Just a suggestion, not a merge-breaker.

If you have established different network segments or subnets for remote worker nodes as described in the section on "Establishing communication between subnets", you must specify the subnets in the `machineNetwork` configuration setting if the workers are using static IP addresses, bonds or other advanced networking. When setting the node IP address in the `networkConfig` paramter for each remote worker node, you must also specify the gateway and the DNS server for the subnet containing the control plane nodes when using static IP addresses. This ensures the remote worker nodes can reach the subnet containing the control plane nodes and that they can receive network traffic from the control plane.

[IMPORTANT]
====
All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details. 

> Deploying a cluster with multiple subnets requires using virtual media.

Reviewer: ```suggestion
Deploying a cluster with multiple subnets requires using virtual media, such as `redfish-virtualmedia` and `idrac-virtualmedia`.
```
